components/tyk-pump/values.yaml

```diff
@@ -206,6 +206,17 @@ pump:
     # Tyk Pump svc is disabled by default. Set it to true to enable it.
     enabled: false
 
+    # healthEnabled enables health check service
```

Comment:
Can you move out these fields from service?
I would suggest to add new field for configuration of healthcheck service. People might want to expose just healthcheck service

Reply:
a new field like `healthCheckService`?

Reply:
Works